### ISSUE TITLE
valueerror-images-of-type-float-must-be-between-1-and-1

### ISSUE THREAD

**Issue by simplesmall** [zhangcliff/style-transfer-of-arbitrary-region#1]
When i try to run it on my ubuntu 16.04 LTS and windows10 by anaconda or vscode,it occur some unexpected wrong,after  debug   I   deal  with many of the frustrating but one thing. After it show valueerror-images-of-type-float-must-be-between-1-and-1
i'v  try to read  source code but have  no  harvest, i'v  trace the code  is already  divide `255``
def net(image,mask):
    image = image / 255.0
    conv1 = _conv_layer(image, 32, 9, 1)
    conv2 = _conv_layer(conv1, 64, 3, 2)
    conv3 = _conv_layer(conv2, 128, 3, 2)
    
    conv3 = tf.concat([conv3,mask],3)
    conv3 = _conv_layer(conv3,128,1,1)
    
    resid1 = _residual_block(conv3, 3)
    resid2 = _residual_block(resid1, 3)
    resid3 = _residual_block(resid2, 3)
    resid4 = _residual_block(resid3, 3)
    resid5 = _residual_block(resid4, 3)
    conv_t1 = resize_conv(resid5, 64, 3, 1)
    conv_t2 = resize_conv(conv_t1, 32, 3, 1)
    conv_t3 = _conv_layer(conv_t2, 3, 9, 1, relu=False)
    preds = tf.nn.tanh(conv_t3)
    output = image + preds
    return tf.nn.tanh(output) * 127.5 + 255./2
i  am  shame to ask such a stupid question but i'm not understand the full  process well.I would thanks a lot if you can do me a fever or lead  me how can I  deal  with this whatever.
